jekyll-sitemap.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r!^(test|spec|features)/!)
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "jekyll", "~> 3.3"
+  spec.add_dependency "jekyll", "~> 3.4", ">= 3.4.2"
```

Comment:
What if we leave the Gemspec as it was, but condition the static file test on `Jekyll::VERSION` being `>= 3.4.2`?